invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentStats/DocumentStats.js

```diff
@@ -38,12 +53,21 @@ export default class DocumentStats extends Component {
               <Table.Row>
                 <Table.HeaderCell>past loans</Table.HeaderCell>
                 <Table.HeaderCell>renewals</Table.HeaderCell>
+                <Table.HeaderCell>
+                  average{' '}
+                  <Popup
+                    position="top right"
+                    content="This is the average number of loans per document item"
```

Comment:
I think we should be more explicit here. The point is to say that it is not exact, because the number of items can vary.
`Note that this average is computed with the current number of items ({itemsCount}) for this book and not with the number of items in the selected range dates.`
@equadon @kprzerwa if you have better suggestions with english, please help.

@topless please show this only if the average is more than 0.

Reply:
@ntarocco I have rephrased it a bit. I found the negation of range dates a bit confusing when I read it for first time. ""